core/db/migrate/20150220155706_add_total_on_hand_to_spree_products.rb

```diff
@@ -0,0 +1,10 @@
+class AddTotalOnHandToSpreeProducts < ActiveRecord::Migration
+  def change
+    add_column :spree_products, :count_on_hand, :integer, null: false, default: 0
+    # seed the counts
+    Spree::Variant.all.each { |v| v.update_counter_cache }
```

Comment:
Some stores have variants into the hundreds of thousands. a) Use active job to update and b) use `#find_each`.